HKMP/Util/FsmUtilExt.cs

```diff
@@ -45,6 +45,17 @@ public static class FsmUtilExt {
             return GetAction(fsm, stateName, index) as T;
         }
 
+        /// <summary>
+        /// Gets the first action from a given state by Type.
+        /// </summary>
+        /// <typeparam name="T">The type of the action that extends FsmStateAction.</typeparam>
+        /// <param name="fsm">The FSM instance.</param>
+        /// <param name="stateName">The name of the state.</param>
+        /// <returns>The action from the FSM or null if the action could not be found.</returns>
+        public static T GetFirstAction<T>(this PlayMakerFSM fsm, string stateName) where T : FsmStateAction {
+            return fsm.GetState(stateName).Actions.OfType<T>().First();
```

Comment:
This can throw an error in two ways, invalidating the doc comment. If the given state name does not correspond to a state in the FSM, it will give a NRE. Or if a state has no actions (for some reason) the `.First()` call will throw an `InvalidOperationException`.